kuma/javascript/src/payments/subscription-form.jsx

```diff
@@ -0,0 +1,107 @@
+// @flow
+import * as React from 'react';
+import { useRef, useState } from 'react';
+
+import { getLocale, gettext, Interpolated } from '../l10n.js';
+
+export default function SubscriptionForm() {
+    const locale = getLocale();
+    const subscriptionFormRef = useRef(null);
+    const [subscribeFormDisabled, setSubscribeFormDisabled] = useState(false);
+    const [subscribeButtonEnabled, setSubscribeButtonEnabled] = useState(false);
+
+    const toggleButton = () => {
+        setSubscribeButtonEnabled(!subscribeButtonEnabled);
+    };
+
+    /**
+     * Opens Stripe modal allowing a user to complete their subscription.
+     * @param {Object} event - The form submit event
+     */
+    const submit = event => {
+        event.preventDefault();
+
+        const subscriptionForm = subscriptionFormRef.current;
+        if (subscriptionForm) {
+            const formData = new FormData(subscriptionForm);
+            setSubscribeFormDisabled(true);
+
+            const stripeHandler = window.StripeCheckout.configure({
+                key: window.mdn.stripePublicKey,
+                locale: 'auto',
+                name: 'MDN Web Docs',
+                zipCode: true,
+                currency: 'usd',
+                amount: 500,
+                email: '',
+                token: function(response) {
+                    formData.set('stripe_token', response.id);
+                    subscriptionForm.submit();
+                },
+                closed: function() {
+                    if (!formData.get('stripe_token')) {
+                        setSubscribeFormDisabled(false);
+                    }
+                }
+            });
+
+            stripeHandler.open();
+        }
+    };
+
+    return (
+        <div className="subscriptions-form">
+            <header className="subscriptions-form-header">
+                <h2>
+                    <Interpolated
+                        id={gettext('$5 <perMontSub />')}
+                        perMontSub={<sub>{gettext('/mo')}</sub>}
+                    />
+                </h2>
+            </header>
+            <form
+                ref={subscriptionFormRef}
+                name="subscription-form"
+                method="post"
+                onSubmit={submit}
+                disabled={subscribeFormDisabled ? 'disabled' : ''}
```

Comment:
Use a plain boolean here instead:
```suggestion
                disabled={subscribeFormDisabled}
```